cli_reference/basic_cli_operations.adoc

```diff
@@ -10,34 +10,56 @@
 toc::[]
 
 == Overview
-This topic provides information on some general CLI operations and their syntax.
+This topic provides information on some general CLI operations and their syntax,
+and assumes you have gone through the link:get_started_cli.html[basic setup and
+login] for the CLI.
```

Comment:
[Suggestion]
This topic provides information on some general CLI operations and their syntax. You must link:get_started_cli.html[configure and set up] the CLI before you can perform these operations.

Generally we try not to assume and just state what you must already have done before you can do anything in this topic, if that's the case.